src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -58,12 +61,12 @@ export class BidResult extends React.Component<BidResultProps> {
       )
     } else {
       const bidAgain = SHOW_TIMER_STATUSES.indexOf(this.props.status) > -1
-      let nextBid = this.props.sale_artwork.minimum_next_bid.display
-      if (
-        this.props.status === "RESERVE_NOT_MET" &&
-        this.props.bid.cents > this.props.sale_artwork.minimum_next_bid.cents
-      ) {
-        // make sure nextBid is higher than users current bid when reserve is not met
+      let nextBid
+      if (this.props.suggested_next_bid) {
```

Comment:
Cool! What are the cases where the `suggested_next_bid` won't be present in the context of this screen? (i.e. when will the other case be executed?)

Reply:
haha just commented [above](https://github.com/artsy/emission/pull/1054#discussion_r191564896)